[2025-02-23T15:00:03.789+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-23T15:00:03.829+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: london_weather_pipeline.transform_weather_data scheduled__2025-02-23T14:00:00+00:00 [queued]>
[2025-02-23T15:00:03.846+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: london_weather_pipeline.transform_weather_data scheduled__2025-02-23T14:00:00+00:00 [queued]>
[2025-02-23T15:00:03.847+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-23T15:00:03.869+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): transform_weather_data> on 2025-02-23 14:00:00+00:00
[2025-02-23T15:00:03.877+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=20272) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-23T15:00:03.878+0000] {standard_task_runner.py:72} INFO - Started process 20276 to run task
[2025-02-23T15:00:03.879+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'london_weather_pipeline', 'transform_weather_data', 'scheduled__2025-02-23T14:00:00+00:00', '--job-id', '139', '--raw', '--subdir', 'DAGS_FOLDER/london_weather_pipeline/london_weather_etl.py', '--cfg-path', '/tmp/tmpsrwe3uxx']
[2025-02-23T15:00:03.880+0000] {standard_task_runner.py:105} INFO - Job 139: Subtask transform_weather_data
[2025-02-23T15:00:03.966+0000] {task_command.py:467} INFO - Running <TaskInstance: london_weather_pipeline.transform_weather_data scheduled__2025-02-23T14:00:00+00:00 [running]> on host ee2081ae3a04
[2025-02-23T15:00:04.151+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='london_weather_pipeline' AIRFLOW_CTX_TASK_ID='transform_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-23T14:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-23T14:00:00+00:00'
[2025-02-23T15:00:04.152+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-23T15:00:04.157+0000] {python.py:240} INFO - Done. Returned value was: {'latitude': '51.5074', 'longitude': '-0.1278', 'temperature': 11.3, 'windspeed': 22.5, 'winddirection': 202, 'weathercode': 3}
[2025-02-23T15:00:04.203+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-23T15:00:04.204+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=london_weather_pipeline, task_id=transform_weather_data, run_id=scheduled__2025-02-23T14:00:00+00:00, execution_date=20250223T140000, start_date=20250223T150003, end_date=20250223T150004
[2025-02-23T15:00:04.293+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-23T15:00:04.327+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-23T15:00:04.330+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
